### ISSUE TITLE
2 users with a course certificate, but no cached enrollment

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#3969]
See https://bi.odl.mit.edu/queries/345/source

**Comment by pdpinch** [comment]
@maxliu can you update the query to pull the username and email? It will save me some time.

**Comment by maxliu** [comment]
If you click the link in your first comment, you will see two users in the list.

**Comment by pdpinch** [comment]
It's down to one user now...

**Comment by pdpinch** [comment]
Next step: pull all the data about this learner to see what the certificate was based on. I hope we find a final grade and an order.

**Comment by pdpinch** [comment]
Lots of edge cases here. 

First of all, sometime after we launched the micromasters portal, edX removed MITx/14.73x/2013_Spring from edx.org and stopped returning enrollment information about it. Many courses from 2012 and 2013 used an older course format which was subsequently deprecated and removed.

Also, we offered the first two exam runs (14.73x and 14.740x, exam series "PILOT") as a free pilot. Since we didn't have the coupon system in place yet, we actually modified the code to record $0 purchases for those participating in the pilot. We also allowed everyone in the pilot to make two attempts. 

I wonder if we should retroactively apply a coupon to these purchases, just to make the accounting consistent...
